src/applications/facility-locator/utils/formatServiceName.js

```diff
@@ -11,6 +11,7 @@ export const formatServiceName = service => {
     .replace(/\s+/g, ' ')
     .toLowerCase()
     .replace(/veteran/g, 'Veteran')
+    .replace(/covid19 vaccine/g, 'COVID-19 vaccines')
```

Comment:
Please add a test to src/applications/facility-locator/tests/helpers/formatServiceName.unit.spec.js, thanks.

Reply:
Yes, thank you. I completely forgot to check if this function already has a test file